config/template.rb

```diff
@@ -8,3 +8,5 @@
 apply 'config/environments/development.rb'
 apply 'config/environments/production.rb'
 apply 'config/environments/test.rb'
+
+copy_file 'config/initializers/backtrace_silencers.rb'
```

Comment:
🗒️  This initializer has been removed from the base application. so now I updated to copy file.

Reply:
But why add it back? Do we make use of it in general?

Reply:
Actually, I'm not sure about that. Seems we're modifying this file to make it work with engine structure. This change is from this issue - https://github.com/nimblehq/rails-templates/issues/147

Maybe we can remove it first and keep our eyes on the error. If we still have the issue, we can add it back again. What do you think?

Reply:
Since it might be hard to reproduce the issue (unless we try creating an engine 😅 ), let's keep it. It is likely that part of Rails did not change much in that version and there this is likely still needed.